.circleci/config.yml

```diff
@@ -1,8 +1,8 @@
 version: 2.1
 
 orbs:
-  ruby: circleci/ruby@1
-  browser-tools: circleci/browser-tools@1.4.0
+  ruby: cimg/ruby@1
+  browser-tools: cimg/browser-tools@1.4.0
```

Comment:
Hi,
Did you get this error? If so, is it fixed now?
```
    Cannot find cimg/ruby@1 in the orb registry. Check that the namespace, orb name and version are correct.
    Cannot find cimg/browser-tools@1.4.0 in the orb registry. Check that the namespace, orb name and version are correct.
```
<img width="1031" alt="Screenshot 2023-04-17 at 13 22 42" src="https://user-images.githubusercontent.com/2813357/232483300-c8d34183-e5ed-4011-80ef-2040f182ba4c.png">

https://app.circleci.com/pipelines/github/NCCE/teachcomputing.org

This doc might help https://circleci.com/developer/images/image/cimg/ruby
(it seems from that that we _might_ not need the ruby orb any more: might be work a try 🤷 )

Reply:
The orbs are still called `circleci/ruby@1` etc